File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/42_compito_python_20230116/Piumino.py
from Articolo import Articolo


class Piumino(Articolo):
    def __init__(self, codice, tipo, marca, taglia, peso, modello, costo, sconto):
        super().__init__(codice, tipo, marca, taglia, modello, costo, sconto)
        self._peso = peso

    def __str__(self):
        return super().__str__() + "\t-\t" + str(self._peso) + "\t" + str(self._costo) + "\t" + str(self._sconto)

File: Programming-languages-and-laboratory/Java/Exercises/42_compito_python_20230116/Piumino.java
public class Piumino extends Articolo {
    private int peso;

    public Piumino(int codice, String tipo, String marca, String taglia, int peso, String modello, float costo,
            int sconto_in_percentuale) {
        super(codice, tipo, marca, taglia, modello, costo, sconto_in_percentuale);
        this.peso = peso;
    }

    public String toString() {
        return super.toString() + "\t-\t" + peso + "\t" + costo + "\t" + sconto_in_percentuale;
    }
}